bodhi/server/consumers/masher.py

```diff
@@ -958,12 +958,24 @@ def compose_atomic_trees(self):
                 release['repos']['updates'] = mash_path
 
             # Compose the tree, and raise an exception upon failure
+            notifications.publish(topic="ostree.start",
+                                  msg=dict(tag=tag),
+                                  force=True)
             result = composer.compose(release)
             if result['result'] != 'success':
                 self.log.error(result)
+                notifications.publish(topic="ostree.start",
```

Comment:
though a little risky, we could make it something like ``topic="ostree.%s" % result['result']`` no?

Risky as in: for every new result, we'll need to adjust fedmsg_meta.